src/material-experimental/mdc-slider/slider.ts

```diff
@@ -778,10 +789,22 @@ export class MatSlider
     // would prefer to use "mousedown" as the default, for some reason it does not work (the
     // callback is never triggered).
     if (this._SUPPORTS_POINTER_EVENTS) {
-      this._elementRef.nativeElement.addEventListener('pointerdown', this._layout);
+      this._elementRef.nativeElement.addEventListener(
```

Comment:
This is a memory leak because the same options aren't being passed back in on `removeEventListener`. Also I think the only listeners where this would make a difference are `touchstart` and `pointerdown`.